champions/src/main/java/me/mykindos/betterpvp/champions/champions/commands/KitCommand.java

```diff
@@ -4,15 +4,44 @@
 import com.google.inject.Singleton;
 import me.mykindos.betterpvp.champions.champions.commands.menu.KitMenu;
 import me.mykindos.betterpvp.core.client.Client;
+import me.mykindos.betterpvp.core.client.Rank;
 import me.mykindos.betterpvp.core.command.Command;
+import me.mykindos.betterpvp.core.config.Config;
+import me.mykindos.betterpvp.core.framework.updater.UpdateEvent;
+import me.mykindos.betterpvp.core.utilities.UtilMessage;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
 import org.bukkit.entity.Player;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
 @Singleton
 public class KitCommand extends Command {
 
     @Inject
     private KitMenu kitMenu;
 
+    @Inject
+    @Config(path = "command.kit.kitCooldown", defaultValue = "true")
+    private boolean kitCooldown;
+
+    @Inject
+    @Config(path = "command.kit.kitCooldownHours", defaultValue = "1")
+    private double kitCooldownHours;
+
+    @Inject
+    @Config(path = "command.kit.kitCooldownMinutes", defaultValue = "0")
+    private double kitCooldownMinutes;
+
+    @Inject
+    @Config(path = "command.kit.kitCooldownSeconds", defaultValue = "0")
+    private double kitCooldownSeconds;
```

Comment:
have just 1 config for minutes, and remember that you want the default value to be a double so e.g. 60.0